pkg/osquery/tables/wlan/psrunner.go

```diff
@@ -0,0 +1,104 @@
+package wlan
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+type PowerShell struct {
+	powerShell string
+}
+
+// New create new session
+func New() *PowerShell {
+	ps, _ := exec.LookPath("powershell.exe")
+	return &PowerShell{
+		powerShell: ps,
+	}
+}
+
+func runPos(ctx context.Context, output *bytes.Buffer) error {
+	posh := New()
+	dir, err := ioutil.TempDir("", "nativewifi")
+	if err != nil {
+		return errors.Wrap(err, "creating nativewifi tmp dir")
+	}
+	defer os.RemoveAll(dir)
+
+	outputFile := filepath.Join(dir, "nativewificode.cs")
+	nativeCodeFile, err := os.Create(outputFile)
+	if err != nil {
+		return errors.Wrap(err, "creating file for native wifi code")
+	}
+	// probably not needed, line 34 covers this
+	defer os.Remove(nativeCodeFile.Name())
+
+	_, err = nativeCodeFile.WriteString(nativeWiFiCode)
+	if err != nil {
+		return errors.Wrap(err, "writing native code file")
+	}
+
+	tmpl, err := template.New("command").Parse(getBSSIDCommandTemplate)
+	if err != nil {
+		return errors.Wrap(err, "parsing template")
+	}
+	commandOpts := struct {
+		NativeCodePath string
+	}{NativeCodePath: nativeCodeFile.Name()}
```

Comment:
I wonder if you really need this. It's putting the full path into the powershell `Add-Type`. Any chance it works with a relative path if you set CWD? That would let you skip the whole template parsing

Reply:
I can give it a shot!